### ISSUE TITLE
bug: Replace object with Extensible

### ISSUE THREAD

**Issue by cre8** [openwallet-foundation-labs/sd-jwt-js#182]
some type are using the object type to extend a type like `type DisclosureFrame<T extends object> = Frame<T>;`

This could be the reason for some errors we have in the veramo implementation. One of their devs suggested to use an extensible like
```
interface Extensible {
  [key: string]?: unknown;
}

type DisclosureFrame<T extends Extensible> = Frame<T>;
```

**Comment by lukasjhan** [comment]
I don't think it will affect other parts.
So Let's give it a try. I think it would be okay to distribute it with the next tag and then check how does it go.

**Comment by cre8** [comment]
I addressed it in #183

**Comment by lukasjhan** [comment]
Can we close this issue? @cre8
